index.js

```diff
@@ -0,0 +1,4 @@
+
+const nombreUsuario = prompt('Ingrese su nombre')
+
+alert(`Hola ${nombreUsuario}!`);
```

Comment:
En este caso, me parece que sería mejor que no uses interpolación de strings
```suggestion
alert('Hola + nombreUsuario');
```

Reply:
este es un comentario en toda la linea